packages/eslint-config-airbnb/rules/variables.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     // disallow shadowing of names such as arguments
     'no-shadow-restricted-names': 2,
     // disallow declaration of variables already declared in the outer scope
```

Comment:
would you mind adding a new comment line linking to the rule definition (http://eslint.org/docs/rules/no-shadow.html)?